### PR TITLE
Fixed compilation errors for latest versions of cmake and boost

### PR DESCRIPTION
**Pull request by gstavrinos**

### CODE REVIEW

File: ka/mutablestore.hpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #define KA_MUTABLESTORE_HPP
 #pragma once
 #include <functional>
-#include <boost/config/suffix.hpp>
+#include <boost/config/detail/suffix.hpp>
```
Comment (nyibbang) [inline ka/mutablestore.hpp]: This file should not be included, could you try removing it completely and see if it compiles for you ?
```suggestion
```

Comment (gstavrinos) [reply ka/mutablestore.hpp]: It actually does, so I removed it. Thanks.

**Comment by nyibbang** [comment]
Seems like commit 2c0d66e is empty and 3fca1f3 and ae149ba are almost the same. Did a merge happen ? Could you try squashing it so we have something linear to merge  ?

**Comment by gstavrinos** [comment]
Oops, sorry. That's what I tried to do with the last merge, but apparently failed. I think the latest commit is fine.

**Pull request merged**